docs/ldml/tr35-numbers.md

```diff
@@ -578,6 +578,20 @@ If there is an explicit negative subpattern, it serves only to specify the negat
 
 > **Note:** The thousands separator and decimal separator in patterns are always ASCII ',' and '.'. They are substituted by the code with the correct local values according to other fields in CLDR. The same is true of the - (ASCII minus sign) and other special characters listed above.
 
+A currency decimal pattern normally contains a currency symbol placeholder (one to four ¤ characters). The currency symbol placeholder may occur before the first digit, after the last digit symbol, or where the decimal symbol would otherwise be placed.
+
+Before | After | Decimal
+-------|-------|-------
+¤#,##0.00|#,##0.00¤|#,##0¤00
+¤ #,##0.00|#,##0.00 ¤|
+¤-#,##0.00|#,##0.00-¤|
+¤ -#,##0.00|#,##0.00- ¤|
+-¤#,##0.00|#,##0.00¤-|
+-¤ #,##0.00|#,##0.00 ¤-|
+…|…|
+
+> If a decimal pattern is used as a currency pattern, and that pattern does not contain a currency symbol placeholder, then it is interpreted as if the decimal symbol were replaced by a single currency symbol. That is, "#,##0.00" ⟹ "#,##0¤00" and "#,##0" ⟹ "#,##0¤". This facility is used for formats such as "12€50", as in "12€50 pour une omelette".
```

Comment:
I don't know if the PR review is the right place to raise this concern, but I'm less and less comfortable with UTS 35 having all these weird edge cases documented in prose. If you don't read this "aside", then you aren't spec-compliant, and you might render monetary values that have no currency symbol! That seems dangerous. I would rather *require* that currency patterns contain `¤`, and permit them to occur at the decimal.

Reply:
Note: This would be a noisy failure for old implementations that don't know how to handle decimal currencies, but that's by design, and better than the alternative of silently failing to display a currency symbol.

Reply:
Ok. I was trying to carry out the intent of your ticket, but went too far here. Will fix.